gtsam/geometry/Pose3.cpp

```diff
@@ -106,7 +106,7 @@ Vector6 Pose3::adjointTranspose(const Vector6& xi, const Vector6& y,
 void Pose3::print(const string& s) const {
   cout << s;
   R_.print("R:\n");
-  cout << '[' << t_.x() << ", " << t_.y() << ", " << t_.z() << "]\';";
+  cout << t_ << ";" << endl;
```

Comment:
Why, what's the reason?

Reply:
Point3 already support ostream so it didn't make much sense to have this duplication of code. I like keep it DRY.

Reply:
Meaning, output is exactly the same before and after?

Reply:
Yes. Verifiably.

Reply:
Good news is, if we change the printing of Point3 in the future, this will be picked up py Pose3 automatically now.